src/helm/benchmark/annotation_executor.py

```diff
@@ -122,3 +131,14 @@ def process(self, annotator_specs: List[AnnotatorSpec], state: RequestState) ->
         except Exception as e:
             raise AnnotationExecutorError(f"{str(e)} Request: {state.request}") from e
         return replace(state, annotations=annotations)
+
+    def process_all(self, annotator_specs: List[AnnotatorSpec], states: List[RequestState]) -> List[RequestState]:
+        annotations: Dict[str, Any] = {}
+        try:
+            for annotator_spec in annotator_specs:
+                annotator: Annotator = self.factory.get_annotator(annotator_spec)
+                new_annotations = annotator.annotate_all(states)
+                annotations[annotator.name] = new_annotations
+        except Exception as e:
+            raise AnnotationExecutorError(f"{str(e)} Request: {states.request}") from e
+        return [replace(state, annotations=annotations) for state in states]
```

Comment:
Doesn't seem right - this code seems to set the same annotation across all instances? You probably need to unpack the scores for all the instances from the response.

Reply:
changed to map the instance level annotations to the request states.